### PR TITLE
Fix regressions after landing drag-n-drop tabs functionality and other minor cleanups

### PR DESCRIPTION
**Pull request by victorporof**
First two commits are renamings and busywork that I've been meaning to land for weeks.

The following commits are regression fixes and other improvements to drag-n-drop.

### CODE REVIEW

**Comment by victorporof** [comment]
@ericawright The problems fixed are the following:
* Pages reloading when tabs are reordered
* Tabs favicons and close buttons overflowing or being larger than they used to
* Tabs separators being invisible
* Pages created via cmd+click not animated when dragged
* Animations were jerky when reordering from left to right (react recreating elements in this case).

Additional stuff:
* Select tab before re-odering.
* Prevent dragging outside of the tabs list container bounds.

**Comment by ericawright** [comment]
thanks for looking after this, I'm PTO today, I'll take a look at it tomorrow

File: src/browser-frontend/views/browser/chrome/tabbar/tab.css [hunk 1]
```diff
@@ -22,7 +22,7 @@ specific language governing permissions and limitations under the License.
   background-position: var(--theme-tab-inactive-background-position);
   background-repeat: var(--theme-tab-background-repeat);
   background-size: var(--theme-tab-background-size);
-  box-shadow: 1px 0 0 0 var(--theme-tab-separator-color);
+  box-shadow: -1px 0 0 0 var(--theme-tab-separator-color);
```
Comment (ericawright) [inline src/browser-frontend/views/browser/chrome/tabbar/tab.css]: is there a reason to take away the tab divider?

**Comment by ericawright** [comment]
the sliding animation looks great, and the reload no longer happens.

one big one:
After closing a tab I get an error `closeTabAnimated TypeError: Cannot read property 'get' of undefined` which then causes most interactions to break.

**Comment by victorporof** [comment]
@ericawright whoops! thanks for looking into it

**Comment by victorporof** [comment]
@ericawright Fixed the issue.

**Review by ericawright — APPROVED** [review]
I can't seem to break anything!

**Pull request merged**